modules/shortcodes.php

```diff
@@ -13,6 +13,11 @@
  * Additional Search Queries: shortcodes, shortcode, embeds, media, bandcamp, dailymotion, facebook, flickr, google calendars, google maps, google+, polldaddy, recipe, recipes, scribd, slideshare, slideshow, slideshows, soundcloud, ted, twitter, vimeo, vine, youtube
  */
 
+if ( ! defined( 'JETPACK_SHORTCODES_PLUGIN_URL' ) ) {
+	define( 'JETPACK_SHORTCODES_PLUGIN_URL', 'modules/shortcodes/' );
```

Comment:
Should we be hard-coding `modules/` here? It's being used below to enqueue the file, but that directory is not found in wpcom